### DISCUSSION TITLE
How to enable community license in my Flutter App ?

### DISCUSSION THREAD

**Discussion by Goldy98** [syncfusion/flutter-examples#423] [Q&A]
Hi ! I'm new in flutter development, and I installed some of syncfusion package from the pub.dev site. 
I came across the DISCLAIMER, and I want to know to attributes the license for the widgets I used in my app

**Comment by AlphaSoundZ** [comment]
[https://help.syncfusion.com/flutter/installation/web-installer/how-to-install](https://help.syncfusion.com/flutter/installation/web-installer/how-to-install)

This should help you out
